compliance/aws/disallowed_regions/aws_disallowed_regions.pt

```diff
@@ -173,21 +173,16 @@ end
 ###############################################################################
 
 escalation "esc_email" do
+  automatic true
   label "Send Mail"
   description "Sends incident email"
   email $param_email
 end
 
 escalation "esc_terminate_instances_disallowed_region" do
-  request_approval do
-    label "Approve Termination of Instances in Disallowed Region"
-    description "Approve escalation to run RightScale Cloud Workflow to terminate instances in disallowed region"
-    parameter "approval_reason" do
-      type "string"
-      label "Reason for Approval"
-      description "Explain why you are approving the action"
-    end
-  end
+  automatic false
+  label "Approve Termination of Instances in Disallowed Region"
```

Comment:
Label should be a simple description one or two words.  How about "Delete Instances'